kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowFragment.kt

```diff
@@ -63,25 +62,38 @@ import io.reactivex.Observable
 @ExperimentalWorkflowUi
 abstract class WorkflowFragment<InputT, OutputT : Any> : Fragment() {
 
-  data class Config<InputT, OutputT : Any>(
+  data class Configured<InputT, OutputT : Any> internal constructor(
```

Comment:
Nit: past tense seems like an odd choice for this name, I kinda liked `Config` better but i don't care much at all.

Reply:
Agreed.